CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Changelog
 
 ### 4.0.0 (Next)
+* [#176](https://github.com/alexa-js/alexa-app/pull/176): Added flexibility to attach alexa-app to express app or router - [@bgjehu](https://github.com/bgjehu).
```

Comment:
@dblock you probably have the answer now. But just to keep it as reference. Yes, the router still need to be attach to an express app. But that should not be in the scope of alexa-app nor alexa.express() function